components/processing/form_parser/src/main.py

```diff
@@ -52,10 +53,12 @@ def batch_process_documents(
         gcs_output_uri: GCS directory to store the out json files,
         gcs_input_prefix: GCS directory to store input files to be processed
     """
+  USER_AGENT = "cloud-solutions/eks-docai"
```

Comment:
add version number

Reply:
@anuradha-bajpai-google - Can you make the suggested changes and submit a PR against version1.2?